jekyll/_cci2/creating-orbs.md

```diff
@@ -6,39 +6,36 @@ description: "Starting point for Creating CircleCI Orbs"
 categories: [getting-started]
 order: 1
 ---
-
+## Introduction
 Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be [used across multiple projects]({{ site.baseurl }}/2.0/using-orbs/). 
 
 [Orbs]({{ site.baseurl }}/2.0/orb-intro/) are made available for use in a configuration through the `orbs` key in the top level of your 2.1 [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file.
 
-## Orb Publishing Process
-
-To publish an orb, follow the steps listed below as an org Admin.
-
-1. Claim a namespace (assuming you don't yet have one), using the command `circleci namespace create <NAMESPACEID> <VCS> <YOURORG>`. For example:
-`circleci namespace create sandbox github CircleCI-Public`
+### Orb Publishing Process
 
-2. Create the orb inside your namespace, for example:
-`circleci orb create sandbox/hello-world`
+Before working with orbs, you may find it helpful to gain a high-level understanding the end-to-end orbs publishing process. The diagram shown below illustrates the orbs publishing process.
```

Comment:
> high-level understanding _of_ the

Reply:
Is this picture available somewhere for review?

Reply:
Hi @hannahhenderson . For some reason right now, the figure is not being displayed. Here is where you can find the diagram:

https://github.com/circleci/circleci-docs/blob/master/jekyll/assets/img/docs/orbsworkflow-v2.png

Reply:
"of" is still missing